### PR TITLE
Tune the intro

### PR DESCRIPTION
**Pull request by artemkovalyov**
## What Has Changed?

Explain what you have changed.

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)

### CODE REVIEW

File: docs/overview/about.mdx [hunk 1]
```diff
@@ -18,7 +18,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ## Introduction
 
-The SAP Cloud SDK is a versatile set of libraries and tools for developers to build applications in a cloud-native way and host them on SAP Business Technology Platform or other runtimes.
+The SAP Cloud SDK is a versatile set of libraries and tools for developers to build applications in a cloud-native way and deliver them via the SAP Business Technology Platform.
+Our goal is to simplify the life of developers working in the SAP ecosystem by abstracting the complexities of the SAP BTP and letting them keep the focus on the application's business logic.
+
+We continuously improve the SAP Cloud SDK in short release cycles to keep up with the enormous speed of changes in cloud development.
+By choosing the SAP Cloud SDK you also chose a carefully selected set of standards and best development practices allowing for a fail-safe, agile, and streamlined software delivery process.
```
Comment (artemkovalyov) [inline docs/overview/about.mdx:25]:
```suggestion
By choosing the SAP Cloud SDK you also chose a curated set of standards and best development practices allowing for a fail-safe, agile, and streamlined software delivery process.
```

File: docs/overview/about.mdx [hunk 2]
```diff
@@ -18,7 +18,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ## Introduction
 
-The SAP Cloud SDK is a versatile set of libraries and tools for developers to build applications in a cloud-native way and host them on SAP Business Technology Platform or other runtimes.
+The SAP Cloud SDK is a versatile set of libraries and tools for developers to build applications in a cloud-native way and deliver them via the SAP Business Technology Platform.
+Our goal is to simplify the life of developers working in the SAP ecosystem by abstracting the complexities of the SAP BTP and letting them keep the focus on the application's business logic.
+
+We continuously improve the SAP Cloud SDK in short release cycles to keep up with the enormous speed of changes in cloud development.
+By choosing the SAP Cloud SDK you also chose a curated set of standards and best development practices allowing for a fail-safe, agile, and streamlined software delivery process.
+To make sure you get a kick start into developing wiht the SAP Cloud SDK we relentlessly update this documentation and run [community events](../community/community-call.mdx).
```
Comment (artemkovalyov) [inline docs/overview/about.mdx:26]:
```suggestion
To make sure you get a kick start into developing with the SAP Cloud SDK we relentlessly update this documentation and run [community events](../community/community-call.mdx).
```

File: docs/overview/about.mdx [hunk 3]
```diff
@@ -18,7 +18,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ## Introduction
 
-The SAP Cloud SDK is a versatile set of libraries and tools for developers to build applications in a cloud-native way and host them on SAP Business Technology Platform or other runtimes.
+The SAP Cloud SDK is a versatile set of libraries and tools for developers to build applications in a cloud-native way and deliver them via the SAP Business Technology Platform.
```
Comment (florian-richter) [inline docs/overview/about.mdx:21]:
```suggestion
The SAP Cloud SDK is a versatile set of libraries and tools for developers to build cloud-native applications on the SAP Business Technology Platform (BTP).
```

File: docs/overview/about.mdx [hunk 4]
```diff
@@ -18,7 +18,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+Our goal is to simplify the life of developers working in the SAP ecosystem by abstracting the complexities of the SAP BTP and letting them keep the focus on the application's business logic.
```
Comment (florian-richter) [inline docs/overview/about.mdx:22]:
```suggestion
Our goal is to abstracting the complexities of the SAP BTP and letting developers focus on the application's business logic.
```

File: docs/overview/about.mdx [hunk 5]
```diff
@@ -18,7 +18,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+We continuously improve the SAP Cloud SDK in short release cycles to keep up with the enormous speed of changes in cloud development.
+By choosing the SAP Cloud SDK you also chose a curated set of standards and best development practices allowing for a fail-safe, agile, and streamlined software delivery process.
```
Comment (florian-richter) [inline docs/overview/about.mdx:25]: For me "fail-safe" means having a plan B but I think here you meant resilient to failure so I suggest to replace it with resilient (or another word that makes this clearer)
```suggestion
By choosing the SAP Cloud SDK, you also choose a curated set of standards and development best practices allowing for a resilient, agile, and streamlined software delivery process.
```

File: docs/overview/about.mdx [hunk 6]
```diff
@@ -18,7 +18,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+To make sure you get a kick start into developing with the SAP Cloud SDK we relentlessly update this documentation and run [community events](../community/community-call.mdx).
```
Comment (florian-richter) [inline docs/overview/about.mdx:26]: Relentless means "unceasingly intense" (as defined by google) which is not wrong but may sound too aggressive. I think regularly gives the same meaning without the aggressive tone (but also a bit weaker promise...)

```suggestion
To make sure you get a kick start into developing with the SAP Cloud SDK, we regularly update this documentation and run [community events](../community/community-call.mdx).
```

File: docs/overview/about.mdx [hunk 7]
```diff
@@ -18,7 +18,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+Welcome on board! Yours, Cloud SDK Team.
```
Comment (florian-richter) [inline docs/overview/about.mdx:28]:
```suggestion
Welcome on board! 
```
or if the sign off is necessary, put it on a separate line.

**Review by florian-richter — CHANGES_REQUESTED** [review]
Shortened a few sentences and added some suggestions for alternative words. Let me know if you disagree with any of the feedback.

File: docs/overview/about.mdx (hunk 4 above)
Comment (artemkovalyov) [reply docs/overview/about.mdx:22]:
```suggestion
Our goal is to simplify the life of developers working in the SAP ecosystem.
We're abstracting the complexities of the SAP BTP and letting you keep the focus on the application's business logic.
```

**Comment by artemkovalyov** [comment]
> Shortened a few sentences and added some suggestions for alternative words. Let me know if you disagree with any of the feedback.

Thanks, @florian-richter , nice ideas. I updated based on your proposals. Take another look.

File: docs/overview/about.mdx [hunk 8]
```diff
@@ -18,7 +18,15 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ## Introduction
 
-The SAP Cloud SDK is a versatile set of libraries and tools for developers to build applications in a cloud-native way and host them on SAP Business Technology Platform or other runtimes.
+The SAP Cloud SDK is a versatile set of libraries and tools for developers to build cloud-native applications on the SAP Business Technology Platform (BTP).
+Our goal is to simplify the life of developers working in the SAP ecosystem.
+We're abstracting the complexities of the SAP BTP and letting you keep the focus on the application's business logic.
```
Comment (florian-richter) [inline docs/overview/about.mdx:23]:
```suggestion
We're abstracting the complexities of the SAP BTP and letting you focus on the application's business logic.
```

**Review by florian-richter — APPROVED** [review]
Tiny improvement proposal below. Otherwise it reads much better now. As a new user, I'd feel welcome and well informed!

**Pull request merged**